roles/openshift_expand_partition/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 ---
 - name: Ensure growpart is installed
   yum: pkg=cloud-utils-growpart state=present
+  when: not is_containerized | bool
+
+- name: Determine if growpart is installed
+  command: "rpm -q cloud-utils-growpart"
+  register: has_growpart
+  failed_when: "has_growpart.cr != 0 and 'package cloud-utils-growpart is not installed' not in has_growpart.stdout"
+  changed_when: false
+  when: is_containerized | bool
```

Comment:
@detiber I actually don't know if we're using it. I'll leave that to @wshearn.

I'd prefer we use cloud-init as well.